Can't view usage of items in my custom pack. Version 2.3.24-GTNH

I'm using NEI Version 2.3.24-GTNH in my custom pack which is mostly based on GTNH, but I can't view the usage of any item. Viewing recipes works fine though. Every time I press the usage key ('U') I get the following message in the chat "There was error during loading recipe. Please check log for details."


I probably missed to provide some vital information, if you need more information please tell me what you need and I'll try to provide the requested information.


The following stacktraces appear in the log:
```
[21:44:59] [ForkJoinPool-1-worker-15/INFO] [NotEnoughItems]: Error parsing CSV
[21:44:59] [ForkJoinPool-1-worker-15/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: java.lang.IllegalArgumentException: Index for header 'modName' is 1 but CSVRecord only has 1 values!
[21:44:59] [ForkJoinPool-1-worker-15/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at codechicken.nei.shadow.org.apache.commons.csv.CSVRecord.get(CSVRecord.java:129)
[21:44:59] [ForkJoinPool-1-worker-15/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at codechicken.nei.recipe.GuiRecipeTab.loadHandlerInfo(GuiRecipeTab.java:236)
[21:44:59] [ForkJoinPool-1-worker-15/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at ru.timeconqueror.tcneiadditions.nei.AspectCombinationHandler.loadUsageRecipes(AspectCombinationHandler.java:51)
[21:44:59] [ForkJoinPool-1-worker-15/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at codechicken.nei.recipe.TemplateRecipeHandler.loadUsageRecipes(TemplateRecipeHandler.java:425)
[21:44:59] [ForkJoinPool-1-worker-15/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at codechicken.nei.recipe.TemplateRecipeHandler.getUsageHandler(TemplateRecipeHandler.java:551)
[21:44:59] [ForkJoinPool-1-worker-15/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at codechicken.nei.recipe.TemplateRecipeHandler.getUsageAndCatalystHandler(TemplateRecipeHandler.java:575)
[21:44:59] [ForkJoinPool-1-worker-15/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at codechicken.nei.recipe.GuiUsageRecipe.getUsageOrCatalystHandler(GuiUsageRecipe.java:54)
[21:44:59] [ForkJoinPool-1-worker-15/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at codechicken.nei.recipe.GuiUsageRecipe.lambda$openRecipeGui$0(GuiUsageRecipe.java:17)
[21:44:59] [ForkJoinPool-1-worker-15/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
[21:44:59] [ForkJoinPool-1-worker-15/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1384)
[21:44:59] [ForkJoinPool-1-worker-15/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:482)
[21:44:59] [ForkJoinPool-1-worker-15/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:472)
[21:44:59] [ForkJoinPool-1-worker-15/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.stream.ReduceOps$ReduceTask.doLeaf(ReduceOps.java:747)
[21:44:59] [ForkJoinPool-1-worker-15/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.stream.ReduceOps$ReduceTask.doLeaf(ReduceOps.java:721)
[21:44:59] [ForkJoinPool-1-worker-15/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.stream.AbstractTask.compute(AbstractTask.java:327)
[21:44:59] [ForkJoinPool-1-worker-15/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:731)
[21:44:59] [ForkJoinPool-1-worker-15/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
[21:44:59] [ForkJoinPool-1-worker-15/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
[21:44:59] [ForkJoinPool-1-worker-15/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
[21:44:59] [ForkJoinPool-1-worker-15/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:175)
```

and 

```
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: java.util.concurrent.ExecutionException: java.lang.NullPointerException
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.concurrent.ForkJoinTask.get(ForkJoinTask.java:1006)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at codechicken.nei.recipe.RecipeHandlerQuery.getHandlersWithRecipes(RecipeHandlerQuery.java:62)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at codechicken.nei.recipe.RecipeHandlerQuery.getRecipeHandlersParallel(RecipeHandlerQuery.java:48)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at codechicken.nei.recipe.RecipeHandlerQuery.runWithProfiling(RecipeHandlerQuery.java:35)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at codechicken.nei.recipe.GuiUsageRecipe.openRecipeGui(GuiUsageRecipe.java:20)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at codechicken.nei.api.ShortcutInputHandler.handleKeyEvent(ShortcutInputHandler.java:46)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at codechicken.nei.recipe.RecipeItemInputHandler.lastKeyTyped(RecipeItemInputHandler.java:15)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at codechicken.nei.guihook.GuiContainerManager.lastKeyTyped(GuiContainerManager.java:397)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at codechicken.nei.recipe.GuiRecipe.func_73869_a(GuiRecipe.java:351)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at net.minecraft.client.gui.inventory.GuiContainer.public_func_73869_a(GuiContainer.java)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at codechicken.nei.guihook.GuiContainerManager.callKeyTyped(GuiContainerManager.java)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at codechicken.nei.guihook.GuiContainerManager.keyTyped(GuiContainerManager.java:570)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at codechicken.nei.guihook.GuiContainerManager.handleKeyboardInput(GuiContainerManager.java:557)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at net.minecraft.client.gui.inventory.GuiContainer.func_146282_l(GuiContainer.java)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at net.minecraft.client.gui.GuiScreen.func_146269_k(GuiScreen.java:276)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1640)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:973)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:5104)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at net.minecraft.client.main.Main.main(SourceFile:148)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.lang.reflect.Method.invoke(Method.java:498)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.lang.reflect.Method.invoke(Method.java:498)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:210)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:245)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at org.multimc.EntryPoint.main(EntryPoint.java:34)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: Caused by: java.lang.NullPointerException
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.concurrent.ForkJoinTask.getThrowableException(ForkJoinTask.java:598)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.concurrent.ForkJoinTask.get(ForkJoinTask.java:1005)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	... 32 more
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: Caused by: java.lang.NullPointerException
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.concurrent.ForkJoinTask.getThrowableException(ForkJoinTask.java:598)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.concurrent.ForkJoinTask.reportException(ForkJoinTask.java:677)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.concurrent.ForkJoinTask.invoke(ForkJoinTask.java:735)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.stream.ReduceOps$ReduceOp.evaluateParallel(ReduceOps.java:714)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:233)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:566)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at codechicken.nei.recipe.RecipeHandlerQuery.lambda$getHandlersWithRecipes$2(RecipeHandlerQuery.java:61)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.concurrent.ForkJoinTask$AdaptedCallable.exec(ForkJoinTask.java:1424)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:175)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: Caused by: java.lang.NullPointerException
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at net.minecraft.item.ItemStack.func_77969_a(ItemStack.java:356)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.stream.MatchOps$1MatchSink.accept(MatchOps.java:90)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.Spliterators$ArraySpliterator.tryAdvance(Spliterators.java:958)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:126)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:499)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:486)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:472)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.stream.MatchOps$MatchOp.evaluateSequential(MatchOps.java:230)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.stream.MatchOps$MatchOp.evaluateSequential(MatchOps.java:196)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.stream.ReferencePipeline.anyMatch(ReferencePipeline.java:516)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at alkalus.main.nei.NEI_Handler_Kettle.loadUsageRecipes(NEI_Handler_Kettle.java:45)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at codechicken.nei.recipe.TemplateRecipeHandler.loadUsageRecipes(TemplateRecipeHandler.java:425)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at codechicken.nei.recipe.TemplateRecipeHandler.getUsageHandler(TemplateRecipeHandler.java:551)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at codechicken.nei.recipe.TemplateRecipeHandler.getUsageAndCatalystHandler(TemplateRecipeHandler.java:575)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at codechicken.nei.recipe.GuiUsageRecipe.getUsageOrCatalystHandler(GuiUsageRecipe.java:54)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at codechicken.nei.recipe.GuiUsageRecipe.lambda$openRecipeGui$0(GuiUsageRecipe.java:17)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1384)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:482)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:472)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.stream.ReduceOps$ReduceTask.doLeaf(ReduceOps.java:747)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.stream.ReduceOps$ReduceTask.doLeaf(ReduceOps.java:721)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.stream.AbstractTask.compute(AbstractTask.java:327)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:731)
[21:44:59] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	... 4 more
```

My modlist is the following:


```
[✔️] 00gtnhmixins
  [📁] 1.7 (folder)
  [✔️] AdvancedSolarPanel-1.7.10-3.5.1a
  [✔️] ae2fc-1.7.10-1.0.43-gtnh
  [✔️] ae2stuff-1.7.10-0.5.1.15-GTNH
  [✔️] aether-1.7.10-v1.1.2.2
  [✔️] AFSU-1.7.10-1.2.6-GTNH
  [✔️] AgriCraft-1.7.10-1.5.0
  [✔️] AppleCore-1.7.10-3.2.3
  [✔️] appliedenergistics2-1.7.10-rv3-beta-130-GTNH
  [✔️] Aquaculture-1.7.10-1.2.6.21
  [✔️] ArchitectureCraft-1.7.10-1.7.6
  [✔️] AsieLib-1.7.10-0.5.3
  [✔️] Automagy-1.7.10-0.28.2
  [✔️] Avaritia-1.7.10-1.31
  [✔️] Avaritiaddons-1.7.10-1.5.4-GTNH
  [✔️] backpack-1.7.10-2.2.8-GTNH
  [✔️] bartworks-1.7.10-0.5.131
  [✔️] Baubles-1.7.10-1.0.1.14
  [✔️] bdlib-1.7.10-1.9.8-GTNH
  [✔️] BeeBetterAtBees-0.3
  [✔️] BetterAchievements-1.7.10-0.1.1
  [✔️] BetterBuildersWands-1.7.10-0.9.3-GTNH
  [✔️] bettercrashes-1.7.10-1.1.7-GTNH
  [✔️] BetterFoliage-MC1.7.10-2.0.9
  [✔️] betterp2p-1.7.10-1.0.12
  [✔️] BiblioCraft[v1.11.7][MC1.7.10]
  [✔️] BiblioWoods[BiomesOPlenty][v1.9]
  [✔️] BiblioWoods[Forestry][v1.7]
  [✔️] BigReactors-0.4.3A
  [✔️] binnie-mods-1.7.10-2.0.46
  [✔️] BiomesOPlenty-1.7.10-2.1.0.2308-universal
  [✔️] BlockLimiter-1.7.10-0.55
  [✔️] blockrenderer6343-1.7.10-1.0.0-RC6
  [✔️] BloodArsenal-1.7.10-1.2.7
  [✔️] BloodMagic-1.7.10-1.3.19
  [✔️] Botania-1.7.10-1.9.11-GTNH
  [✔️] BrandonsCore-1.7.10-1.0.0.13-GTNH
  [✔️] bugtorch-1.7.10-1.1.6.5-GTNH
  [✔️] buildcraft-1.7.10-7.1.28
  [✔️] buildcraft-compat-1.7.10-7.1.12
  [✔️] BuildCraftOilTweak-1.7.10-1.0.3
  [✔️] campfirebackport-1.7.10-1.10
  [📁] carpentersblocks (folder)
  [✔️] CarpentersBlocks-1.7.10-3.3.8.2-GTNH
  [✔️] catwalks-1.7.10-2.1.3-GTNH
  [✔️] ChestTransporter-1.7.10-2.0.6
  [✔️] chisel-1.7.10-2.10.16-GTNH
  [✔️] chiseltones-1.7.10-1.0.4-GTNH
  [✔️] CodeChickenCore-1.7.10-1.1.6
  [✔️] CoFHCore-[1.7.10]3.1.4-329
  [✔️] CompactKineticGenerators-1.7.10-1.0.1
  [✔️] ComputerCraft1.75
  [✔️] Computronics-1.7.10-1.6.17-GTNH
  [✔️] controlling-1.7.10-1.0.0.4
  [✔️] CookingForBlockheads-1.7.10-1.2.13-GTNH
  [✔️] CraftingEX_1.7.2-v1.0.5.1
  [✔️] CraftTweaker-1.7.10-3.2.9
  [✔️] creativecore-1.7.10-1.3.27-GTNH
  [✔️] CropLoadCore-1.7.10-0.1.10
  [✔️] CropsPP-1.7.10-1.3.6.11
  [✔️] Draconic-Evolution-1.7.10-1.1.7-GTNH
  [✔️] DummyCore-1.7.10-1.15
  [✔️] EasyCrafting-1.7.10-2.0.1.16
  [✔️] EMT-1.7.10-1.2.8.25
  [✔️] EnchantingPlus-1.7.10-3.0.2-d
  [✔️] endercore-0.2.12
  [✔️] EnderIO-1.7.10-2.3.1.49
  [✔️] EnderIO-1.7.10-2.3.1.51
  [✔️] EnderStorage-1.7.10-1.4.12
  [✔️] Et_Futurum_Requiem-2.4.2
  [✔️] eternalsingularity-1.7.10-1.0.5
  [✔️] ExtraCells-1.7.10-2.5.29
  [✔️] extrautilities-1.2.13a
  [✔️] Fancy+Fluid+Storage-1.7.10-1.3.7
  [✔️] fastcraft-1.25
  [✔️] fastflyblockbreaking-MC1.7.10-1.1
  [✔️] findit-1.7.10-1.0.11
  [✔️] FloodLights-1.7.10-1.2.9
  [✔️] Forbidden.Magic-1.7.10-0.6.4-GTNH
  [✔️] Forestry-1.7.10-4.5.6
  [✔️] forgelin-1.7.10-1.9.6-GTNH
  [✔️] ForgeRelocation-1.7.10-0.0.3
  [✔️] ForgeRelocationFMP-1.7.10-0.0.4
  [✔️] FTBLib-1.7.10-1.0.18.3
  [✔️] FTBUtilities-1.7.10-1.0.18.3
  [✔️] funky-locomotion-1.7.10-beta-7
  [✔️] gadomancy-1.7.10-1.0.8.5
  [✔️] galacticgreg-1.7.10-1.0.8
  [✔️] Galacticraft-1.7.10-3.0.60-GTNH
  [✔️] GalaxySpace-1.7.10-1.1.23-GTNH
  [✔️] gendustry-1.7.10-1.6.5.5-GTNH
  [✔️] ggfab-1.7.10-0.0.9
  [✔️] GoodGenerator-1.7.10-0.4.72
  [✔️] GraviSuite-1.7.10-2.0.68u
  [✔️] gregtech-1.7.10-5.09.41.184
  [✔️] GT.Scanner.Mod-1.7.10-1.6.9
  [✔️] gtneioreplugin-1.7.10-1.0.25
  [✔️] GTNewHorizonsCoreMod-1.7.10-1.9.117
  [✔️] GTNewHorizonsCoreMod-1.7.10-1.9.142-pre
  [✔️] gtnhlanth-1.7.10-0.9.23
  [✔️] gtnhlib-1.7.10-0.0.8.1
  [✔️] GT-PlusPlus-1.7.10-1.7.172
  [✔️] GTTweaker-1.7.10-1.7.5
  [✔️] harvestcraft-1.7.10-1.0.19-GTNH
  [✔️] Healer-1.2.0
  [✔️] HelpFixer-1.7.10-1.1.0
  [✔️] hodgepodge-1.7.10-2.0.11
  [✔️] holoinventory-1.7.10-2.1.19-GTNH
  [✔️] HungerOverhaul-1.7.10-1.0.4-GTNH
  [✔️] hydroenergy-1.7.10-1.0.16
  [📁] ic2 (folder)
  [✔️] IC2 Crop Plugin Version 1.7.10-1.3.1
  [✔️] IC2NuclearControl-1.7.10-2.4.16
  [✔️] IguanaTweaksTConstruct-1.7.10-2.2.3
  [✔️] ImmersiveEngineering-0.7.7
  [✔️] industrialcraft-2-2.2.828a-experimental
  [✔️] InGameInfoXML-1.7.10-2.8.3.93
  [✔️] INpureCore-1.7.10-1.1.2-GTNH
  [✔️] inventorytweaks-1.7.10-1.5.15
  [✔️] IronChest-1.7.10-6.0.71
  [✔️] IronChestMinecarts-1.7.10-1.0.8
  [✔️] irontanks-1.7.10-1.2.6
  [✔️] itlt-1.7.10-1.0.4
  [✔️] JABBA-1.7.10-1.2.21
  [✔️] journeymap-1.7.10-5.1.4p2-unlimited
  [✔️] KeepingInventory-1.7.10-2.1
  [✔️] kekztech-1.7.10-0.6.24
  [✔️] kubatech-1.7.10-0.7.2
  [✔️] littletiles-1.7.10-1.2.7-GTNH
  [✔️] logisticspipes-1.7.10-0.9.4.5.9-GTNH
  [✔️] LunatriusCore-1.7.10-1.1.7-GTNH
  [✔️] magicbees-1.7.10-2.5.15-GTNH
  [✔️] Mantle-1.7.10-0.3.5
  [✔️] Mekanism-1.7.10-9.1.1.1031
  [✔️] MekanismGenerators-1.7.10-9.1.1.1031
  [✔️] MineFactoryReloaded-[1.7.10]2.8.2B1-201
  [✔️] MineTweakerRecipeMaker-1.7.10-1.1.0.11
  [✔️] mixingasm-1.7.10-v0.2.3
  [✔️] ModTweaker2-1.7.10-0.9.9
  [✔️] modularui-1.7.10-1.0.31
  [✔️] Morpheus-1.7.10-1.6.21
  [✔️] MouseTweaks-1.7.10-2.4.6-GTNH
  [✔️] mxrandom-1.7.10-0.1.5
  [✔️] naturescompass-1.7.10-1.3.6-GTNH
  [✔️] NEIAddons-1.7.10-1.12.22
  [✔️] NEICustomDiagram-1.7.10-1.4.4.6
  [✔️] NEIIntegration-1.7.10-1.2.3
  [✔️] netherportalfix-mc1.7.10-1.1.1
  [✔️] NettyPatch-1.7.10-0.3.1
  [✔️] NodalMechanics-1.7.10-1.1.-6-GTNH
  [✔️] NoMoreRecipeConflict-1.7.10-0.3.2
  [✔️] notenoughIDs-1.7.10-1.4.4
  [✔️] NotEnoughItems-1.7.10-2.3.18-GTNH
  [✔️] NotEnoughItems-2.3.24-GTNH
  [✔️] NotEnoughResources-1.7.10-0.1.0-128
  [✔️] oauth-1.7.10-1.06.1-GTNH
  [✔️] OpenBlocks-1.7.10-1.6.9-GTNH
  [✔️] OpenComputers-1.7.10-1.8.0.2-GTNH
  [✔️] OpenGlasses-1.7.10-1.2.5-GTNH
  [✔️] OpenModsLibs-1.7.10-0.10.6
  [✔️] OpenModularTurrets-1.7.10-2.2.11-247
  [✔️] openprinter-1.7.10-0.1.2-GTNH
  [✔️] Opis-1.7.10-1.3.7-mapless
  [✔️] OreExcavation-1.1.134
  [✔️] OreTweaker-1.0.2-1.7.10
  [✔️] overloadedarmorbar-1.7.10-1.0.1
  [✔️] personalspace-1.7.10-1.0.19
  [✔️] PlayerAPI-1.7.10-1.4.2
  [✔️] ProjRed-1.7.10-4.7.9-GTNH
  [✔️] Railcraft-1.7.10-9.13.14
  [✔️] RandomThings-2.2.4
  [✔️] RemoteIO-1.7.10-2.4.7
  [✔️] roguelike-1.7.10-1.5.3-GTNH
  [✔️] Schematica-1.7.10-1.9.1-GTNH
  [✔️] SGCraft-1.7.10-1.3.13-GTNH
  [✔️] sleepingbag-1.7.10-0.1.4
  [✔️] Snad-1.7.10-1.6.01.31a
  [✔️] SpiceOfLife-1.7.10-2.0.7-carrot
  [✔️] SSP-1.1.2-GT-NH-Mod
  [✔️] StevesAddons-1.7.10-0.10.24
  [✔️] StevesCarts-1.7.10-2.0.2
  [✔️] StevesFactoryManager-1.7.10-1.1.1-GTNH
  [✔️] StorageDrawers-1.7.10-1.11.19-GTNH
  [✔️] StorageDrawersBop-1.7.10-1.11.17-GTNH
  [✔️] StorageDrawersForestry-1.7.10-1.11.17-GTNH
  [✔️] StorageDrawersMisc-1.7.10-1.11.17-GTNH
  [✔️] structurecompat-1.7.10-0.3.0
  [✔️] structurelib-1.7.10-1.2.0-beta.2
  [✔️] SuperTic-1.7.10-1.2.5
  [✔️] Tainted-Magic-r7.6
  [✔️] tcinventoryscan-1.7.10-1.0.12-GTNH
  [✔️] tcneiadditions-1.7.10-1.1.1.7.dirty
  [✔️] tcnodetracker-1.7.10-1.1.6
  [✔️] TConstruct-1.7.10-1.9.8-GTN
  [✔️] TecTech-1.7.10-5.0.61
  [✔️] TGregworks-1.7.10-GTNH-1.0.23
  [✔️] Thaumcraft-1.7.10-4.2.3.5a
  [✔️] Thaumcraft4Tweaks-1.4.26
  [✔️] ThaumcraftMobAspects-1.7.10-1.0.0-GTNH
  [✔️] thaumcraftneiplugin-1.7.10-1.7a
  [✔️] ThaumcraftResearchTweaks-1.7.10-1.0.4
  [✔️] Thaumic Machina-1.7.10-0.2.1
  [✔️] Thaumic-Based-1.7.10-1.4.31
  [✔️] thaumicenergistics-1.7.10-1.3.24-GTNH
  [✔️] Thaumic-Exploration-1.7.10-1.1.92-GTNH
  [✔️] ThaumicHorizons-1.7.10-1.2.1.8
  [✔️] thaumicinsurgence-1.7.10-0.0.8
  [✔️] ThaumicTinkerer-1.7.10-2.6.5
  [✔️] ThermalDynamics-[1.7.10]1.2.1-172
  [✔️] ThermalExpansion-[1.7.10]4.1.5-248
  [✔️] ThermalFoundation-[1.7.10]1.2.6-118
  [✔️] TiCTooltips-1.7.10-1.2.10
  [✔️] Translocator-1.7.10-1.1.2.20
  [✔️] TravellersGear-1.7.10-1.16.8-GTNH
  [✔️] TwilightForest-1.7.10-2.3.8.17
  [✔️] txloader-1.7.10-1.1
  [✔️] Universal-Singularities-1.7.10-8.6.5
  [✔️] VeganOption-mc1.7.10-0.2.2
  [✔️] visualprospecting-1.7.10-1.0.36
  [✔️] Waila-1.7.10-1.5.22
  [✔️] WailaHarvestability-1.7.10-1.1.10-GTNH
  [✔️] WAILAPlugins-1.7.10-0.2.5
  [✔️] WanionLib-1.7.10-1.8.3
  [✔️] WarpTheory-1.7.10-1.2.7-GTNH
  [✔️] Wawla-1.7.10-1.1.3-GTNH
  [✔️] Waystones-mc1.7.10-1.0.12
  [✔️] WirelessCraftingTerminal-1.7.10-1.8.9.2
  [✔️] witchery-1.7.10-0.24.1
  [✔️] WitcheryExtras-1.7.10-1.1.12
  [✔️] WitchingGadgets-1.7.10-1.2.21-GTNH
  [✔️] World-Stripper-1.0.0-1.7.10
  [✔️] WR-CBE-1.7.10-1.4.8
  [✔️] YAMCore-1.7.10-0.5.84
  [✔️] Ztones-1.7.10-2.2.2
```

--- a/src/main/java/alkalus/main/nei/NEI_Handler_Cauldron.java
+++ b/src/main/java/alkalus/main/nei/NEI_Handler_Cauldron.java
@@ -49,7 +49,7 @@ public class NEI_Handler_Cauldron extends TemplateRecipeHandler {
     public void loadCraftingRecipes(final ItemStack result) {
         for (final BrewActionRitualRecipe ritual : WitcheryBrewRegistry.INSTANCE.getRecipes()) {
             for (final BrewActionRitualRecipe.Recipe recipe : ritual.getExpandedRecipes()) {
-                if (result.isItemEqual(recipe.result)) {
+                if (NEIServerUtils.areStacksSameTypeCrafting(result, recipe.result)) {
                     this.arecipes.add(new CachedKettleRecipe(recipe.result, recipe.ingredients, getPowerCost(ritual)));
                 }
             }
@@ -61,7 +61,7 @@ public class NEI_Handler_Cauldron extends TemplateRecipeHandler {
             try {
                 for (final BrewActionRitualRecipe.Recipe recipe : ritual.getExpandedRecipes()) {
                     for (final ItemStack stack : recipe.ingredients) {
-                        if (stack.isItemEqual(ingredient)) {
+                        if (NEIServerUtils.areStacksSameTypeCrafting(stack, ingredient)) {
                             this.arecipes.add(
                                     new CachedKettleRecipe(recipe.result, recipe.ingredients, getPowerCost(ritual)));
                         }

--- a/src/main/java/alkalus/main/nei/NEI_Handler_Kettle.java
+++ b/src/main/java/alkalus/main/nei/NEI_Handler_Kettle.java
@@ -49,7 +49,8 @@ public class NEI_Handler_Kettle extends TemplateRecipeHandler {
 
     public void loadUsageRecipes(final ItemStack ingredient) {
         for (final KettleRecipes.KettleRecipe recipe : KettleRecipes.instance().recipes) {
-            if (Arrays.stream(recipe.inputs).anyMatch(ingredient::isItemEqual)) {
+            if (Arrays.stream(recipe.inputs)
+                    .anyMatch(item -> NEIServerUtils.areStacksSameTypeCrafting(item, ingredient))) {
                 this.arecipes.add(new CachedKettleRecipe(recipe.output, recipe));
             }
         }

